index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Addisalem seifu Portofolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <p>
+                My Logo
+            </p>
+            <img src="./img/menue.png" alt="">
+        </nav>
+    </header>
+    <section>
+        <div class="details-section">
+            <h1>I'm Addisalem am Very Happy to see you</h1>
+            <p class="details">i am a software developer by profesion and i help develop many softwares, you can go ahed and check out some of my projects in my portofolio, both mobile and web based and continuing to do so. if you need help regarding software development or have a project in mind feel free to contact me.</p>
```

Comment:
Add line break before and after  this paragraph to avoid the overlapping shown below
![wrong](https://user-images.githubusercontent.com/58376020/125995165-f57f0eab-8467-48ff-9c6d-ddaccf5d7285.png)